src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,15 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine[] machines = new Machine[]{truck, bulldozer, excavator};
```

Comment:
let's inline this code and make it more concise. Introducing a new variables is not needed if its used only one time.

Reply:
Okay, but i don't understand how i must improve this code because i create these variables to pass them to an array